lib/prom/prom.go

```diff
@@ -0,0 +1,121 @@
+package prom
+
+import (
+	"fmt"
+	"net"
+	"net/http"
+	"time"
+
+	"github.com/gorilla/mux"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	vegeta "github.com/tsenart/vegeta/v12/lib"
+)
+
+//PrometheusMetrics vegeta metrics observer with exposition as Prometheus metrics endpoint
+type PrometheusMetrics struct {
```

Comment:
Since the package name is already `prom`, I think we can get rid of the `Prometheus` prefix everywhere, so that it reads `prom.Metrics`

Reply:
@flaviostutz: Pending.

Reply:
done